exercises/wordy/.meta/template.j2

```diff
@@ -0,0 +1,20 @@
+{%- import "generator_macros.j2" as macros with context -%}
+{{ macros.header() }}
+
+class {{ exercise | camel_case }}Test(unittest.TestCase):
+    {% for case in cases -%}
+    def test_{{ case["description"] | to_snake }}(self):
+        value = "{{ case["input"]["question"] }}"
+        {%- if case is error_case %}
+        with self.assertRaisesWithMessage(ValueError):
+            {{ case["property"] }}(value)
+        {%- else %}
+        expected = {{ case["expected"] }}
+        self.assertEqual({{ case["property"] }}(value), expected)
+        {%- endif %}
+
+    {% endfor %}
+
+    {{ macros.utility() }}
+
+{{ macros.footer() }}
```

Comment:
`has_error_case` is a built-in variable for the generator that is `True` if there is any test case that expects an error. The `footer` macro takes it as an optional parameter.

```suggestion
{{ macros.footer(has_error_case) }}
```

Reply:
This one, I copied it from the hamming exercise. To stick with this advice, we could create an issue and PR for that ?

Reply:
There's actually a larger issue that footer macro shouldn't need that variable to be passed in, but removing it will break some existing templates, so that's a larger fix.